AElf.Kernel/Storages/ChangesStore.cs

```diff
@@ -1,24 +1,64 @@
-﻿using System.Threading.Tasks;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AElf.Kernel.Extensions;
 
 namespace AElf.Kernel.Storages
 {
     public class ChangesStore : IChangesStore
     {
-        private readonly IKeyValueDatabase _keyValueDatabase;
+        private readonly KeyValueDatabase _keyValueDatabase;
+        private readonly KeyValueDatabase _keyValueDatabaseForPaths;
 
-        public ChangesStore(IKeyValueDatabase keyValueDatabase)
+        private static readonly Hash HashToGetPaths = new Hash("paths".CalculateHash());
+
+        public ChangesStore(KeyValueDatabase keyValueDatabase, KeyValueDatabase keyValueDatabaseForPaths)
         {
             _keyValueDatabase = keyValueDatabase;
+            _keyValueDatabaseForPaths = keyValueDatabaseForPaths;
+            _keyValueDatabaseForPaths.SetAsync(HashToGetPaths, new List<Hash>());
```

Comment:
@loning I'm not sure whether List<Hash> can be serialized by protobuf, but for now the implementation need to get all the changes to generate a WorldState instance, thus I use another KeyValueDatabase to store the hash list. So can you help to review this implementation?

Reply:
you can define a protobuf message with repeat filed

Reply:
see BlcokBody -> transactions